### PR TITLE
Nf core template merge 2.0.1

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/strelka/germline/main.nf [hunk 1]
```diff
@@ -19,36 +19,36 @@ process STRELKA_GERMLINE {
     }
 
     input:
-    tuple val(meta), path(cram), path(crai)
+    tuple val(meta), path(bam), path(bai)
```
Comment (FriederikeHanssen) [inline modules/local/strelka/germline/main.nf:22]: goign back to bam here...

File: modules/local/strelka/germline/main.nf [hunk 2]
```diff
@@ -19,36 +19,36 @@ process STRELKA_GERMLINE {
...
     path  fasta
     path  fai
-    tuple path(target_bed), path(target_bed_tbi)
+    path  target_bed
```
Comment (FriederikeHanssen) [inline modules/local/strelka/germline/main.nf:25]: us target_bed containing the tbi thing as well?

File: modules/nf-core/modules/samtools/merge/main.nf [hunk 3]
```diff
@@ -22,8 +22,8 @@ process SAMTOOLS_MERGE {
     tuple val(meta), path(bams)
 
     output:
-    tuple val(meta), path("${prefix}.bam"), emit: merged_bam
-    path  "*.version.txt" ,                 emit: version
+    tuple val(meta), path("${prefix}.bam"), emit: bam
```
Comment (FriederikeHanssen) [inline modules/nf-core/modules/samtools/merge/main.nf:25]: also back to bams here...I think it is jsut a naming thing. I guess, that is how they come from nf-core/modules

**Review by FriederikeHanssen — APPROVED** [review]
Just the one thing about bams. Other than that looks good :)

File: modules/nf-core/modules/samtools/merge/main.nf (hunk 3 above)
Comment (maxulysse) [reply modules/nf-core/modules/samtools/merge/main.nf:25]: actually we have the samtools/merge_cram module in local, we probably don't need both, I'll check this out.
Thanks for the review

Comment (maxulysse) [reply modules/nf-core/modules/samtools/merge/main.nf:25]: ok, so we need both for now, but it'll be nice to have only one module, I'll look this up in a separate PR

**Pull request merged**